.travis.yml

```diff
@@ -6,13 +6,21 @@ python:
     - "3.7"
     - "3.7-dev"
 matrix:
+    include:
+        - os: windows
+          language: sh
+          python: "3.7"
```

Comment:
Should we also include python version 3.6 in this build matrix?

Reply:
Included Python 3.6.8 @subhashb